includes/class-wc-stripe-webhook-handler.php

```diff
@@ -1240,8 +1302,9 @@ protected function defer_webhook_processing( $webhook_notification, $additional_
 	 *
 	 * @param string $webhook_type    The webhook event name/type.
 	 * @param array  $additional_data Additional data passed to the scheduled job.
+	 * @param stdClass $notification  The webhook notification payload.
 	 */
-	public function process_deferred_webhook( $webhook_type, $additional_data ) {
+	public function process_deferred_webhook( $webhook_type, $additional_data, $notification ) {
```

Comment:
I think we should explicitly default `$notification` to `null` to handle cases where the data was not included, e.g. from just before an upgrade.
```suggestion
	public function process_deferred_webhook( $webhook_type, $additional_data, $notification = null ) {
```